src/elements/content-sidebar/activity-feed/app-activity/AppActivity.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../../common/variables';
+
+.be {
+    .bcs-app-activity-container {
+        position: relative;
+        z-index: 1;
+
+        &:hover,
+        &.bcs-is-focused {
+            .bcs-inline-delete-container {
+                opacity: 1;
+            }
+        }
+
+        .bcs-app-activity {
+            display: flex;
+            margin-top: 23px;
+            opacity: .9999;
+            width: 100%;
+
+            &.bcs-is-pending {
+                opacity: .4;
+            }
+
+            .bcs-app-activity-icon {
+                flex-shrink: 0;
+                height: 32px;
+                margin-right: 10px;
+                width: 32px;
+            }
+
+            .bcs-app-activity-content {
+                color: $fours;
+                flex: 1;
+                min-width: 0;
+
+                .bcs-app-activity-link {
```

Comment:
Do these need hover and focus styles? Should they use the `link` class?

Reply:
Already using the `link` class since it's a `Link` component. It requires additional color, as design requested. Note that designs do not outline any different hover behaviour